libs/api/domains/driving-license-book/src/models/drivingLicenseTestResult.resopnse.ts

```diff
@@ -0,0 +1,46 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class DrivingLicenceTestResult {
```

Comment:
Skip suffix

Reply:
This result is referring to test results not result from the query 😄

Reply:
aaah I see 👍